aws/resource_aws_ses_domain_mail_from_test.go

```diff
@@ -0,0 +1,76 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAwsSESDomainMailFrom_basic(t *testing.T) {
+	domain := fmt.Sprintf(
+		"%s.terraformtesting.com",
+		acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(testAccAwsSESDomainMailFromConfig, domain),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsSESDomainMailFromExists("aws_ses_domain_mail_from.test"),
```

Comment:
Sorry I forgot to mention in the initial review -- we should verify the attributes are set correctly in the Terraform state as well 👍  e.g.
```go
resource.TestCheckResourceAttr("aws_ses_domain_mail_from.test", "domain", domain),
resource.TestCheckResourceAttr("aws_ses_domain_mail_from.test", "mail_from_domain", fmt.Sprintf("bounce.%s", domain)),
```